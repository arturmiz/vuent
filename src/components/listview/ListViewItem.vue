<template>
  <div class="vnt-listview-item"
       role="listitem">
    <slot></slot>
  </div>
</template>

<script>
export default {
  name: 'VntListviewItem'
};
</script>

<style lang="scss">
@import '../../scss/mixins/component';
@import '../../scss/mixins/text';

.vnt-listview-item {
  @include component-base;
  @include text-overflow;

  padding: 12px 8px;

  &:active {
    background-color: #f9f9f9;
  }
}
</style>
