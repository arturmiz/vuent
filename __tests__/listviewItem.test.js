import { shallowMount } from '@vue/test-utils';
import ListViewItem from '@/components/listview/ListViewItem.vue';

describe('ListViewItem', () => {
  let wrapper;

  describe('by default', () => {

    beforeAll(() => {
      wrapper = shallowMount(ListViewItem);
    });

    test('renders correctly', () => {
      expect(wrapper).toMatchSnapshot();
    });

  });

});
