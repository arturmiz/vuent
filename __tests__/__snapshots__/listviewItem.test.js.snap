// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListViewItem by default renders correctly 1`] = `<div role="listitem" class="vnt-listview-item"></div>`;
